File: WebApi/InWords.WebApi/InWords.Data.DTO/WordTranslation.cs
namespace InWords.Data.DTO
{
    /// <inheritdoc />
    /// <summary>
    ///     Foreign and Native words pair
    /// </summary>
    public class WordTranslation : SyncBase
    {
        public string WordForeign { get; set; }

        public string WordNative { get; set; }

        #region Ctor

        public WordTranslation()
        {
        }

        public WordTranslation(string wordForeign, string wordNative, int serverId = 0) : base(serverId)
        {
            WordForeign = wordForeign;
            WordNative = wordNative;
        }

        public WordTranslation(WordTranslation wordTranslation) : base(wordTranslation)
        {
            WordForeign = wordTranslation.WordForeign;
            WordNative = wordTranslation.WordNative;
        }

        #endregion
    }
}

File: Android/app/src/main/java/ru/inwords/inwords/translation/domain/model/WordTranslation.kt
package ru.inwords.inwords.translation.domain.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.io.Serializable

@Parcelize
data class WordTranslation(
    val wordForeign: String,
    val wordNative: String,
    val id: Long = 0,
    val serverId: Int = 0
) : Parcelable, Serializable